scripts/commands/zone.lua

```diff
@@ -333,7 +333,7 @@ function onTrigger(player, bytes)
     else
         -- destination is a zone ID.
         zone = tonumber(bytes)
-        if zone == nil or zone < 0 or zone >= xi.zone.MAX_ZONE then
```

Comment:
`>= 300` .. That should have stopped it the same as `> 299` , I'm confused here.

side note: This +1 from the actual highest ID thing is the same pattern that has been used with every MAX_WHATEVER since the old PXI days to, so I suggest checking the rest for essential updates and consistency lets contributors be confused on usage and introduce new errors in the future.

Reply:
300 is an invalid zone ID